kernel/src/capabilities.rs

```diff
@@ -117,3 +117,7 @@ pub unsafe trait CreatePortTableCapability {}
 /// A capsule would never hold this capability although it may hold
 /// capabilities created via this capability.
 pub unsafe trait NetworkCapabilityCreationCapability {}
+
+/// The `SysTickFrequencyCapability` allows the holder to change the Cortex M
+/// SysTick `hertz` field.
+pub unsafe trait SysTickFrequencyCapability {}
```

Comment:
This is only a minor nit, but can you move this capability declaration to the `systick.rs` file itself? In contrast to the other capabilities defined in this file, this one is specific to the particular SysTick driver implementation and so should be tightly coupled to it.

Reply:
Sure Leon, hope it's okay now.